### PR TITLE
WIP: Python test_sql refactor all comparisons to inside SQL

### PR DESCRIPTION
**Pull request by vpipkt**
Hoping that this will pass Travis build.

### CODE REVIEW

**Comment by vpipkt** [comment]
Well that did not work 

https://travis-ci.org/locationtech/rasterframes/builds/566080081#L4979-L5013

**Comment by metasim** [comment]
Added `show` to see what the numbers are:

https://travis-ci.org/locationtech/rasterframes/builds/566536333#L5122-L5133

```
5060
5061+------------------+------------------+------------------+------------------+------------------+
5062|base              |plus_one          |minus_one         |double            |half              |
5110+------------------+------------------+------------------+------------------+------------------+
5111|10573.227770833333|10465.225422680413|10572.227770833333|21146.455541666666|5286.613885416667 |
5112|9672.139422680413 |9673.139422680413 |9671.139422680413 |19344.278845360826|4836.069711340207 |
5113|10122.969770833333|10019.60925773196 |10121.969770833333|20245.939541666667|5061.484885416667 |
5114|9912.923030927835 |9913.923030927835 |9911.923030927835 |19825.84606185567 |4956.461515463918 |
5115|10606.11498969072 |10607.11498969072 |10605.11498969072 |21212.22997938144 |5303.05749484536  |
5116|10488.786144329897|10489.786144329897|10487.786144329897|20977.572288659794|5244.3930721649485|
5117|10305.663113402063|10306.663113402063|10304.663113402063|20611.326226804125|5152.831556701031 |
5118|9605.92806185567  |9606.92806185567  |9604.92806185567  |19211.85612371134 |4802.964030927835 |
5119+------------------+------------------+------------------+------------------+------------------+
```

**Comment by vpipkt** [comment]
Going totally out on a limb. The other odd thing is we now get warnings on pyrsistent python library install. Maybe bump version back to 0.15.3? Note 0.15.4 released Jul 27. I have no idea if/how that might affect these tests. Will try it out and we can back it out if it doesn't help.

**Comment by vpipkt** [comment]
Hmm now no Travis run on the PR 🤦‍♂

File: pyrasterframes/src/main/python/setup.py [hunk 1]
```diff
@@ -128,7 +128,8 @@ def doctype(self):
         'setuptools>=0.8',
         'ipython==6.2.1',
         "ipykernel==4.8.0",
-        'Pweave==0.30.3'
+        'Pweave==0.30.3',
+        'pyrsistent==0.15.3',
```
Comment (vpipkt) [inline pyrasterframes/src/main/python/setup.py]: Probably needs another look if this is needed.

**Pull request closed without merging**